src/Node/CaptureNode.php

```diff
@@ -36,10 +36,10 @@ public function compile(Compiler $compiler): void
         }
         $compiler
             ->raw($useYield ? "implode('', iterator_to_array(" : '\\Twig\\Extension\\CoreExtension::captureOutput(')
-            ->raw("(function () use (&\$context, \$macros, \$blocks) {\n")
+            ->raw("(function () use (&\$context, \$macros, \$blocks): iterable {\n")
```

Comment:
this is not needed, generated code don't need extra runtime checks, let's remove them all from this PR

Reply:
Done!